### PR TITLE
Requirements Checker: add intl extension checking

### PR DESCRIPTION
**Pull request by MartinSadovy**

### CODE REVIEW

File: tools/Requirements-Checker/checker.php [hunk 1]
```diff
@@ -206,6 +206,13 @@
 );
 
 $tests[] = array(
+	'title' => 'Intl extension',
+	'required' => FALSE,
+	'passed' => class_exists('Transliterator', FALSE),
+	'description' => 'Class Transliterator for script transliteration in Nette\Utils\Strings::webalize() and toAscii().',
```
Comment (Majkl578) [inline tools/Requirements-Checker/checker.php]: Can you change the message to match the above ones? Something like _"Class Transliterator is absent, the output of Nette\Utils\Strings::webalize() and Nette\Utils\Strings::toAscii() may not be accurate."_

Comment (dg) [inline tools/Requirements-Checker/checker.php]: …for non-latin alphabets.

Comment (Majkl578) [inline tools/Requirements-Checker/checker.php]: :+1:

Comment (MartinSadovy) [inline tools/Requirements-Checker/checker.php]: :+1: fixed

**Pull request merged**